d2l-input-textarea.html

```diff
@@ -0,0 +1,308 @@
+<!--
```

Comment:
Do we still need these licenses at the beginning of the component, test, and demo files?

Reply:
I don't think so.